fjord/feedback/static/js/generic_feedback.js

```diff
@@ -160,5 +160,8 @@
                 selectHappySad(1);
             }
         }
+        if (qs.url) {
+            $('#id_url').val(qs.url);
```

Comment:
I think if we trigger the input event validation we don't need to validate before pre-filling, since the form will do the validation for us already. I will make it trigger the event.